overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-dracut-rootfs.sh

```diff
@@ -38,7 +50,10 @@ case "${1:-}" in
         ;;
     cleanup)
         if [ -d "${saved_sysroot}" ]; then
+            dev=$(grep "${saved_sysroot}" /proc/mounts | awk -F'[^0-9]*' '{print $2}')
             umount "${saved_sysroot}"
+            echo /dev/zram"${dev}" > /sys/class/zram-control/hot_remove
```

Comment:
Shouldn't this be

```suggestion
            echo ${dev} > /sys/class/zram-control/hot_remove
```

? Or does it work both way?

Reply:
yes you're right